### PR TITLE
Some small things...

### PR DESCRIPTION
**Pull request by bkueng**
@julianoes can you review this please?

### CODE REVIEW

File: src/modules/mavlink/mavlink_main.cpp [hunk 1]
```diff
@@ -1920,7 +1918,6 @@ Mavlink::task_main(int argc, char *argv[])
 		configure_stream("EXTENDED_SYS_STATE", 2.0f);
 		configure_stream("ALTITUDE", 10.0f);
 		configure_stream("VISION_POSITION_NED", 10.0f);
-		configure_stream("NAMED_VALUE_FLOAT", 50.0f);
```
Comment (julianoes) [inline src/modules/mavlink/mavlink_main.cpp]: Nice, makes sense to me.

File: src/modules/sdlog2/sdlog2_messages.h [hunk 2]
```diff
@@ -610,11 +610,11 @@ struct log_PARM_s {
 	char name[16];
 	float value;
 };
+#pragma pack(pop)
 
 // the lower type of initialisation is not supported in C++
 #ifndef __cplusplus
 
-#pragma pack(pop)
```
Comment (julianoes) [inline src/modules/sdlog2/sdlog2_messages.h]: Makes sense.

File: src/platforms/posix/main.cpp [hunk 3]
```diff
@@ -329,13 +329,13 @@ int main(int argc, char **argv)
 				}
 
 				if (buf_ptr_write > 0) {
-					if (mystr != string_buffer[buf_ptr_write - 1]) {
+					if (!mystr.empty() && mystr != string_buffer[buf_ptr_write - 1]) {
```
Comment (julianoes) [inline src/platforms/posix/main.cpp]: Awesome, I had that on my list for a while!

**Comment by julianoes** [comment]
@bkueng nice, good to merge.

**Comment by bkueng** [comment]
@julianoes thanks, just pushed one more fix...

**Comment by julianoes** [comment]
Seems good, valgrind FTW.

**Comment by LorenzMeier** [comment]
Rebased and applied, thanks!

**Comment by bkueng** [comment]
@LorenzMeier where did you apply this? It's not in master...

**Comment by LorenzMeier** [comment]
Yep, I totally failed. I couldn't resurrect the commits from my local copy, I quickly re-did them to not bother you. Thanks for catching this. This is a real issue with the non-Github rebase workflow, since one wrong step during rebase can put a PR onto a stale branch.

**Comment by bkueng** [comment]
Ok thanks, no problem. I still would have had them too.

**Pull request closed without merging**